rollups/evm/single/docker-compose.yml

```
@@ -0,0 +1,71 @@
+
+volumes:
+  logs:
+    driver: local
+  evm-single-data:
+
+networks:
+  rollkit-network:
+    driver: bridge
+    ipam:
+      config:
+        - subnet: 172.16.0.0/24
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)